__tests__/integration/GetAppController.test.ts

```diff
@@ -0,0 +1,24 @@
+import supertest from 'supertest';
+
+import { app } from '../../src/app';
+
+describe('GetAppController', () => {
+  let ghostAgent = supertest.agent(app);
+
+  it('should return 200', async () => {
```

Comment:
Change for the test to be clearer, rename test for "should return statuscode 200"